### DISCUSSION TITLE
Does Blaze support multitenancy?

### DISCUSSION THREAD

**Discussion by florisa** [samply/blaze#635] [Q&A]
to support multiple tenants?

**Comment by alexanderkiel** [comment]
Hi Florisa, unfortunately Blaze doesn't support multitenancy or any kind of authorisation. On the other hand. it's easy to start multiple instances of Blaze. You can change the [CONTEXT_PATH](https://github.com/samply/blaze/blob/9b2e0269d2bef728cef37427dac2ac4c4ebaf411/docs/deployment/environment-variables.md), so you can deploy many instances of Blaze under the same domain. That could be a way, if you only need a small amount of tenants, so that the extra amount of memory stays manageable.
